doc/tutorials/neighbor_search.md

```diff
@@ -418,8 +418,7 @@ The `RuleType` class provides the following functions for use in the traverser:
 // Evaluate the base case between two points.
 double BaseCase(const size_t queryIndex, const size_t referenceIndex);
 
-// Score the two nodes to see if they can be pruned, returning DBL_MAX if they
-// can be pruned.
+// Score the two nodes to see if they can be pruned, returning DBL_MAX
```

Comment:
```suggestion
// Score the two nodes to see if they can be pruned, returning DBL_MAX if so.
```
DBL_MAX isn't always what is returned, it's only what's returned when the nodes can be pruned.